scripts/run_server.py

```diff
@@ -35,34 +38,46 @@ def parse_args():
 
     logging.basicConfig(level=args.log_level.upper())
 
-    simulator_config = SimulatorConfig(
+    # Define a custom list of init agents positions (ugly here but will be done in a yaml file later)
+    if args.custom_pos:
+        coord = 0
+        positions = []
+        # Here all the agents are placed on the box diagonal
+        for i in range(args.n_agents):
+            coord += 10
+            positions.append([coord, coord])
+    else:
+        positions = None
+
+    simulator_state = init_simulator_state(
```

Comment:
This part works to show that we can use a list of coordinates, but I think it should be removed before the merge